lib/dash/dash_parser.js

```diff
@@ -818,7 +815,6 @@ shaka.dash.DashParser = class {
         textStreams: this.periodCombiner_.getTextStreams(),
         imageStreams: this.periodCombiner_.getImageStreams(),
         offlineSessionIds: [],
-        minBufferTime: minBufferTime || 0,
```

Comment:
Now that we are only using `minBufferTime` to calculate the `defaultPresentationDelay` of the presentation timeline, perhaps the code where we read the `minBufferTime` from the manifest (and where we take `ignoreSuggestedPresentationDelay` into account) can be moved into the clause where we make the presentation timeline?
If the part of the code where we get the `minBufferTime` and the part of the code where we use it are right next to each other, it'll improve readability.

Reply:
Done!